augur/refine.py

```diff
@@ -196,6 +197,7 @@ def run(args):
                     Tc=0.01 if args.coalescent is None else args.coalescent, #use 0.01 as default coalescent time scale
                     use_marginal = args.date_inference == 'marginal',
                     branch_length_inference = args.branch_length_inference or 'auto',
+                    precision = args.precision or 'auto',
```

Comment:
@rneher this won't allow `args.precision` to be `0` (not sure if `0` is synonymous with 'auto')